docs/2 Architecture/3 Standard Modules.md

```diff
@@ -58,10 +67,14 @@ _Read the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/x/par
 
 Those modules are typically installed on default when using starport. There are a range of modules that are also part of the Cosmos SDK, additionally some other public modules have already reached a major level of usage and acceptance. We will look at more advanced modules in the next Chapter.
 
-## Summary
+## Using modules
+
+With starport you can add a module with the command `starport add modulename`. When adding a module manually to a blockchain application, it requires to edit the `app/app.go` and the `myappcli/main.go` with the according entries. Starport manages the code edits and additions for you conviniently.
 
-- The basic modules form the foundation of a Cosmos SDK blockchain.
-- The modules allow for account management, sending Tokens, managing supply and blockchain access.
-- Modules are "plug-and-play" for the Cosmos SDK.
+## Summary
 
-[◀️ Previous - What are modules](../../03%20Modules/01_what_are_modules/01_what_are_modules.md) | [▶️ Next - Advanced Modules](../../03%20Modules/03_advanced_modules/03_advanced_modules.md)  
\ No newline at end of file
+- Importing modules in a Cosmos SDK built blockchain exposes new functionality for the blockchain.
+- Any combination of modules is allowed.
```

Comment:
Well, theoretically it depends. I believe you can have only one module that updates the validator set. But I'm fine to let this if we speak about "classic" modules